### ISSUE TITLE
Table of contents colors 

### ISSUE THREAD

**Issue by ricschuster** [NCC-CNC/wheretowork#154]
@jeffreyhanson if I remember correctly you mentioned that having color picking functionality in the table of contents is complicated. Did I get that right? If so, lets leave as is and I just label this as `sometime` for now.

**Comment by jeffreyhanson** [comment]
Yeah that's right - thanks for positing it here so we don't forget about it.

**Comment by ssast** [comment]
Closing for now, but logged as a potential UI enhancement.
